### PR TITLE
fix(jenkins): deprecate `tiup package` usages

### LINKED ISSUE #2542: Deprecation: `tiup package`
0d23c5e2-acef-4dfe-8efe-04559b6aab9d

It can just do with tar command, the tiup package is deprecated.

### PR DESCRIPTION
**Pull request by wuhuizuo**
Fixes #2542

Signed-off-by: wuhuizuo <wuhuizuo@126.com>

### CODE REVIEW

File: jenkins/pipelines/cd/tiup/grafana-tiup-mirror-update-test.groovy [hunk 1]
```diff
@@ -65,16 +65,14 @@ def pack = { version, os, arch ->
     wget -qnc https://raw.githubusercontent.com/pingcap/tiflash/${tag}/metrics/grafana/tiflash_summary.json
 
     cd ..
-    tiup package . -C grafana-${version} --hide --arch ${arch} --os "${os}" --desc 'Grafana is the open source analytics & monitoring solution for every database' --entry "bin/grafana-server" --name grafana --release "${RELEASE_TAG}"
+    tar -czvf package/grafana-${RELEASE_TAG}-${os}-${arch}.tar.gz .
```
Comment (lijie0123) [inline jenkins/pipelines/cd/tiup/grafana-tiup-mirror-update-test.groovy:68]:
```suggestion
    tar -C grafana-${version} -czvf package/grafana-${RELEASE_TAG}-${os}-${arch}.tar.gz .
```

**Review by purelind — COMMENTED** [review]
Why modify this file on current pr  `scripts/pingcap/community/update-prow-owners.ts`

**Comment by wuhuizuo** [comment]
> Why modify this file on current pr `scripts/pingcap/community/update-prow-owners.ts`

it's fixed by rebasing.

**Review by lijie0123 — APPROVED** [review]

**Pull request merged**